src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -42,29 +41,29 @@ public EditCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditCommand.MESSAGE_USAGE), pe);
         }
 
-        argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS);
+        argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_BUDGET);
 
-        EditPersonDescriptor editPersonDescriptor = new EditPersonDescriptor();
+        EditCommand.EditCustomerDescriptor editCustomerDescriptor = new EditCommand.EditCustomerDescriptor();
```

Comment:
I think you do not need to put `EditCommand.EditCustomerDescriptor` as the data type. You can probably get away with `EditCustomerDescriptor`. 😸